include/aspect/simulator.h

```diff
@@ -804,6 +804,16 @@ namespace aspect
        */
       void denormalize_pressure(LinearAlgebra::BlockVector &vector);
 
+      /**
+       * apply the bound preserving limiter to the discontinuous galerkin solutions:
```

Comment:
In principle I like this function, however in perspective we might need to call it for temperature and composition. So maybe you could prepare for that case by also handing over an `AdvectionField` parameter that denotes, which field the limiter to apply to. For now the function can throw an exception as soon as somebody hands over `AdvectionField::temperature()`. With this change we do not need to change the argument list of the function when we can deal with temperature as well. Also please capitalize 'Apply' at the beginning of the line (it will appear in the online documentation at https://aspect.dealii.org/doc/doxygen/classaspect_1_1Simulator.html and it looks nicer that way).